src/Microsoft.AspNetCore.Razor.Language/Legacy/CSharpTokenizer.cs

```diff
@@ -547,11 +547,8 @@ private StateResult VerbatimStringLiteral()
             else if (EndOfFile)
             {
                 CurrentErrors.Add(
-                    RazorDiagnostic.Create(
-                        new RazorError(
-                            LegacyResources.ParseError_Unterminated_String_Literal,
-                            CurrentStart,
-                            length: 1 /* end of file */)));
+                    RazorDiagnosticFactory.CreateParsing_UnterminatedStringLiteral(
+                        new SourceSpan(CurrentStart, contentLength: 1 /* end of file */)));
```

Comment:
Suggestion - turn the `1` into a named constant or static factory on `SourceSpan` like `ToEndOfFile`. Then the comments aren't needed here, the code will be self documenting. This is common enough that it seems worth it.

Reply:
As far as I can see we only do this in 2 places in our source. I don't think it's worth special casing this. I'll leave this as is for now. I'll do it if I find more places where this is needed.